mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module.asciidoc

```diff
@@ -1,37 +1,36 @@
-:important-caption: :heavy_exclamation_mark:
-:caution-caption: :fire:
-:warning-caption: :warning:
-endif::[]
-
-toc::[]
-:idprefix:
-:idseparator: -
-:reproducible:
-:source-highlighter: rouge
-:listing-caption: Listing
-
-= Core Test Module
-
-== What is Core Test Module
-
-image::images/image1_new.png[]
-
-== Core Test Module Functions
-
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Test-reports-with-logs-and-or-screenshots.asciidoc[Test reports with logs and/or screenshots]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-test-groups-tags.asciidoc[Test groups/tags]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Data-driven-approach.asciidoc[Data driven approach]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Test-case-parallel-execution.asciidoc[Test case parallel execution]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-BDD-Gherkin-Cucumber-approach.asciidoc[BDD - Gherkin - Cucumber approach]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Run-on-independent-Operating-Systems.asciidoc[Run on independent Operating Systems]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Externalize-test-environment-DEV-QA-SIT-PROD.asciidoc[Externalize test environment (DEV, QA, SIT, PROD)]
-* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Different-Environments.asciidoc[Encrypting sensitive data]
-
-== How to start?
-
-Read: https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Framework-Test-Class.asciidoc[Framework Test Class]
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
+= Core Test Module
+
+== What is Core Test Module
+
+image::images/image1_new.png[]
+
+== Core Test Module Functions
+
+* https://github.com/devonfw/devonfw-testing/blob/develop/mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Test-reports-with-logs-and-or-screenshots.asciidoc[Test reports with logs and/or screenshots]

```

Comment:
Instead of link to github you can use reference to the document section like this:
* link:Core-Test-Module-Test-reports-with-logs-and-or-screenshots.asciidoc[Test reports with logs and/or screenshots]